src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -111,7 +124,10 @@ private string GenerateResultString(Types.IEvaluation evaluation)
             {
                 foreach (var result in evaluation.Results)
                 {
-                    resultString += $"{TwiceIndentedNewLine}Line: {result.LineNumber}";
+                    if (!result.Passed)
+                    {
+                        resultString += $"{TwiceIndentedNewLine}Line: {result.LineNumber}";
+                    }
```

Comment:
This message is already not printed if the evaluation passed (line 85).  Instead of checking here, I suggest moving the call to `GenerateResultString` inside the `if` statement on line 85.